playbook.yml

```diff
@@ -3,79 +3,87 @@
 - name: Install thirdparties
   hosts: trento-server
   become: true
+  pre_tasks:
+    - name: Check SLES distribution and version
```

Comment:
We need to check if we need to accept openSUSE leap as well.
Otherwise the PR environment workflow will start failing, as It is not based on SLES4SAP (but i'm not sure if it is based in Leap neither)